meta/main.yml

```diff
@@ -0,0 +1,17 @@
+galaxy_info:
+  author: Sean Myers
+  destiption: Manage the creation and destruction of openstack resources
+              for running molecule in a shared CI environment.
+  description: My Description
+  company: Red Hat, Inc.
+  license: GPLv3
+  min_ansible_version: 2.4
+  platforms:
```

Comment:
Maybe this can be for all types of platforms?

Reply:
I don't know how to specify that in meta/main...I'm not sure it's possible without basically poking galaxy API and generating a list of all the platforms it knows. I think I'll just remove the platforms list entirely (it'll default to an empty list).

I'll also add the "molecule" tag.